packages/solid/src/reactive/signal.ts

```diff
@@ -1482,7 +1480,13 @@ function runTop(node: Computation<any>) {
 }
 
 function runUpdates<T>(fn: () => T, init: boolean) {
-  if (Updates) return fn();
+  if (Updates) {
+    try {
```

Comment:
I am not sure about the consequences of doing that, so I would be grateful for guidance here.

Reply:
Did you need to do this? The idea in general here is that the only times Updates exists is if part of a parents update. I guess that is the concern because injected owner should handle it rather than the starter of the updates. You start see why I like doing it in runWithOwner.

Reply:
You are right Ryan :) I can see it now. Let me think about it a little more.